pkg/initializer/initializer.go

```diff
@@ -57,8 +58,15 @@ func (e *EtcdInitializer) Initialize(mode validator.Mode, failBelowRevision int6
 	ctx := context.Background()
 	var err error
 
-	//Etcd cluster scale-up case
-	if miscellaneous.IsMultiNode(logger) {
+	podName, err := miscellaneous.GetEnvVarOrError("POD_NAME")
+	if err != nil {
+		logger.Fatalf("Error reading POD_NAME env var : %v", err)
+	}
+
+	// Etcd cluster scale-up case
+	// Note: first member of etcd cluster can never be part of scale-up case.
+	// TODO: consider removing this special check for first cluster member when backup-restore can check presence of any member in cluster.
+	if miscellaneous.IsMultiNode(logger) && !strings.HasSuffix(podName, "0") {
```

Comment:
```
if !strings.HasSuffix(podName, "0") {
...
```
This is a sufficient condition now. Do you want to consider using this instead?

Reply:
i think it should be fine. Besides, this condition clearly denotes that the cluster is multinode *and* the pod is not `pod-0`. Although just `!pod-0` would suffice (because any pod not having suffix `0` would mean it's a multinode cluster), it reduces readability, especially since this is more of a temporary fix and we're not looking to optimise code execution here but rather keep the fix readable in order to remove the technical debt later.

Reply:
>  it reduces readability, especially since this is more of a temporary fix and we're not looking to optimise code execution here but rather keep the fix readable in order to remove the technical debt later.


agreed, we can go ahead as it reduces the readability